source/game.roomhandler.adagency.bmx

```diff
@@ -890,29 +826,29 @@ Type RoomHandler_AdAgency Extends TRoomHandler
 		Next
 		'=== LOWEST ===
 		levelFilters[0] = New TAdContractbaseFilter
-		'from 80-120% of lowest (Minimum of 0.01%)
-		levelFilters[0].SetAudience(Max(spotMin, lowestChannelQuoteDaytime), Max(spotMin , averageChannelQuoteDayTime))
+		'from lowest to average day quote (Minimum of 0.01%)
+		levelFilters[0].SetAudience(Max(spotMin, d.lowestChannelQuoteDaytime), Max(spotMin , d.averageChannelQuoteDayTime))
 		'1% - avgImage %
-		levelFilters[0].SetMinImageRange(0.0, lowestChannelImage)
+		levelFilters[0].SetMinImageRange(0.0, d.lowestChannelImage)
 		'lowest should be without "limits"
 		levelFilters[0].SetSkipLimitedToProgrammeGenre()
 		levelFilters[0].SetSkipLimitedToTargetGroup()
 		If limitInstances > 0 Then levelFilters[0].SetCurrentlyUsedByContractsLimit(0, limitInstances-1)
 
 		'=== AVERAGE ===
 		levelFilters[1] = New TAdContractbaseFilter
-		'from 80% of avg to lowest prime quote, may cross with lowest!
-		levelFilters[1].SetAudience(Max(spotMin, 0.8 * averageChannelQuoteDayTime), Max(spotMin, lowestChannelQuotePrimeTime))
+		'from 80% of avg to lowest prime/dayaverage, may cross with lowest and high
+		levelFilters[1].SetAudience(Max(spotMin, 0.8 * d.averageChannelQuoteDayTime), Max(d.highestChannelQuoteDayTime, d.lowestChannelQuotePrimeTime))
 		'0-100% of average Image
-		levelFilters[1].SetMinImageRange(0, averageChannelImage)
+		levelFilters[1].SetMinImageRange(0, d.averageChannelImage)
 		If limitInstances > 0 Then levelFilters[1].SetCurrentlyUsedByContractsLimit(0, limitInstances-1)
 
 		'=== HIGH ===
 		levelFilters[2] = New TAdContractbaseFilter
-		'from 50% of avg to 150% of highest
-		levelFilters[2].SetAudience(Max(spotMin, 0.8 * highestChannelQuoteDayTime), Max(spotMin, 1.5 * highestChannelQuotePrimeTime))
+		'from 80% of highest day to 120% of highest prime
```

Comment:
Nur als Seitennotiz: die 150% waren dem "Ansporn" geschuldet. Es wurde sich damals beschwert, dass der Werbemakler nur Werbung anbot, die "ein leichtes" fuer das folgende Primetime-Programm war (dank Ausbau nach Mitternacht und somit nach "Statistikerhebung").
Eventuell sollte also die Statistikerhebung mehrfach am Tag erfolgen (bzw "invalidiert" werden ... allerdings bedeutet dies, dass er erste "nach Invalidierung" den Makler nutzende Spieler das dann "genaueste" Angebot bekommt)

Reply:
Mehrfache Statistikerhebung am Tag würde nur Sinn ergeben, wenn die aktuellen Zahlen verwendet werden und nicht nur die des Vortages (ich hatte auch über "laufende" Aktualisierung der Statisik nach Ausstrahlung nachgedacht, halte das aber aktuell nicht für sinnvoll). Diese Anpassung ist ja in dem Filter, der seit langem standardmäßig gar nicht verwendet wird.
Ich schau mir die Werte nochmal in einem KI-Spiel an.

Reply:
Ich denke, der Faktor ist nicht ganz unproblematisch. Ich hatte die Anpassung gemacht, da der Faktor 1.2 für Refill2 verwendet wurde (Angleichung der oberen Grenze für beide Varianten). Für Refill1 habe ich nun 1.3 eingetragen. 
Für den ersten Tag ist der Wert illusorisch (bis zu 210K Zuschauer - das dürfte nur in Ausnahmefällen zu schaffen sein). Für spät im Spiel (prozentual kleine Zuwächse durch Zukauf) dürfte es auch ausreichend sein. Betroffen dürfte wohl die Anfangsausbauphase sein, wo man durch Zukauf eine prozentual starke Erhöhung der Zuschauerzahl erreicht. Auch ist es ja so, dass die Zuschaueranforderung der Werbung mit gestiegener Reichweite zunimmt. Leicht zu schaffen bezieht sich also ggf. auf schon geholte Werbung.

Die aktuelle Berechnung basiert ja nun auf den wirklichen Zuschauerzahlen. Mittelfristig könnte man sich ja auch bei Reichweiteänderungen informieren lassen und den oberen Faktor variieren je nachdem wie groß der prozentuale Unterschied zur vorigen Reichweite ist (je größer, desto höher der Faktor). Für jetzt würde ich aber erstmal die Reaktionen bezüglich der "grundsätzlichen" Anpassung abwarten. Aktuelle Beschwerde ist ja eher, dass die Werbung für den Tag fehlt (nur kleine und Prime-Werbung). Und mit diesem PR soll ja vorrangig dieser Punkt adressiert werden.